### ISSUE TITLE
Problem with last page when finalising the proposal

### ISSUE THREAD

**Issue by iron76** [pdebuyl/FET_Proposal#1]
There seems to be a problem with the last page counter when compiling `final.tex`. The last page is zeroed even if it works fine when compiling `project.tex`.

**Comment by pdebuyl** [comment]
Could you provide more details on how you compiled?

**Comment by iron76** [comment]
Just did the test to be sure. Freshly cloned from your repository (not from my forked version). I am simply running `make -B final` and the resulting `final.pdf` has pages labeled `page 2 of 0`, `page 3 of 0` and so on.

**Comment by pdebuyl** [comment]
Hi, the problem also occurs for examples of https://github.com/KWARC/LaTeX-proposal/
For the draft it is ok. I have tried to debug this but it goes beyond my LaTeX knowledge. @kohlhase are you aware of this problem (would fix upstream if knew what to do)?

**Comment by iron76** [comment]
@pdebuyl may be it is a good idea to open issue on https://github.com/KWARC/LaTeX-proposal/

**Comment by kohlhase** [comment]
indeed, that would be helpful. You have to format the proposal twice.

**Comment by pdebuyl** [comment]
Even several compilations do not help. I opened an issue at https://github.com/KWARC/LaTeX-proposal/issues/14
